app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java

```diff
@@ -80,6 +81,8 @@ public class LoginActivity extends AccountAuthenticatorActivity {
     final String saveErrorMessage ="errorMessage";
     final String saveUsername="username";
     final  String savePassword="password";
+    final String loginUsername = "loginUsername";
+    final String loginMessage = "loginMessage";
```

Comment:
Is it not possible to make this a public final constant and reuse the same in the CommonsApplication's logout listener?